### PR TITLE
 [Samples][Java] Fix disparities of 02.echo-bot sample

### PR DESCRIPTION
**Pull request by VictorGrycuk**
Fixes #

## Proposed Changes

  - 
  -
  -

### CODE REVIEW

File: samples/java_springboot/02.echo-bot/README.md [hunk 1]
```diff
@@ -15,26 +17,26 @@ This sample is a Spring Boot app and uses the Azure CLI and azure-webapp Maven p
   - Build the sample using `mvn package`
   - Run it by using `java -jar .\target\bot-echo-sample.jar`
 
-- Test the bot using Bot Framework Emulator
-
-  [Bot Framework Emulator](https://github.com/microsoft/botframework-emulator) is a desktop application that allows bot developers to test and debug their bots on localhost or running remotely through a tunnel.
+## Testing the bot using Bot Framework Emulator
+[Bot Framework Emulator](https://github.com/microsoft/botframework-emulator) is a desktop application that allows bot developers to test and debug their bots on localhost or running remotely through a tunnel.
 
   - Install the Bot Framework Emulator version 4.3.0 or greater from [here](https://github.com/Microsoft/BotFramework-Emulator/releases)
```
Comment (Batta32) [inline samples/java_springboot/02.echo-bot/README.md]: Check the [sample 16](https://github.com/microsoft/BotBuilder-Samples/tree/Java-R13/samples/java_springboot/16.proactive-messages) for this. We are not indicating to install 4.3.0 or greater

File: samples/java_springboot/02.echo-bot/README.md [hunk 2]
```diff
@@ -54,10 +56,14 @@ Record the `appid` from the returned JSON
 Replace the values for `<appid>`, `<appsecret>`, `<botname>`, and `<groupname>` in the following commands:
 
 #### To a new Resource Group
-`az deployment sub create --name "echoBotDeploy" --location "westus" --template-file ".\deploymentTemplates\template-with-new-rg.json" --parameters appId="<appid>" appSecret="<appsecret>" botId="<botname>" botSku=S1 newAppServicePlanName="echoBotPlan" newWebAppName="echoBot" groupLocation="westus" newAppServicePlanLocation="westus"` 
+```
+az deployment sub create --name "proactiveBotDeploy" --location "westus" --template-file ".\deploymentTemplates\template-with-new-rg.json" --parameters appId="<appid>" appSecret="<appsecret>" botId="<botname>" botSku=S1 newAppServicePlanName="proactiveBotPlan" newWebAppName="proactiveBot" groupLocation="westus" newAppServicePlanLocation="westus"
+```
```
Comment (Batta32) [inline samples/java_springboot/02.echo-bot/README.md]: Be careful, you pasted the proactive things

File: samples/java_springboot/02.echo-bot/README.md [hunk 3]
```diff
@@ -54,10 +56,14 @@ Record the `appid` from the returned JSON
...
 
 #### To an existing Resource Group
-`az deployment group create --resource-group "<groupname>" --template-file ".\deploymentTemplates\template-with-preexisting-rg.json" --parameters appId="<appid>" appSecret="<appsecret>" botId="<botname>" newWebAppName="echoBot" newAppServicePlanName="echoBotPlan" appServicePlanLocation="westus" --name "echoBot"`
+```
+az deployment group create --resource-group "<groupname>" --template-file ".\deploymentTemplates\template-with-preexisting-rg.json" --parameters appId="<appid>" appSecret="<appsecret>" botId="<botname>" newWebAppName="proactiveBot" newAppServicePlanName="proactiveBotPlan" appServicePlanLocation="westus" --name "proactiveBot"
+```
```
Comment (Batta32) [inline samples/java_springboot/02.echo-bot/README.md]: same

**Review by Batta32 — CHANGES_REQUESTED** [review]

File: samples/java_springboot/02.echo-bot/README.md (hunk 3 above)
Comment (VictorGrycuk) [reply samples/java_springboot/02.echo-bot/README.md]: Updated both commands replacing the proactive reference with echo

**Review by Batta32 — APPROVED** [review]

**Pull request closed without merging**